docs/Reference/Orion-CLI-Syntax.md

```diff
@@ -41,6 +41,14 @@ Displays help and exits.
 
 Displays version information and exits.
 
+## clear known nodes
+
+```bash tab="Syntax"
+  --clear-known-nodes
```

Comment:
@NicolasMassart - I'm not sure exactly what you mean by this?  These are CLI options so they will all come after the `orion` command.  Am I misunderstanding what you mean?

Reply:
I think this is because I had this option prefixed with `##` instead of `###`. By making it clear it's part of the options, it should hopefully make it visually more part of the options block, which doesn't prepend the command as part of its description.

Reply:
I added the extra `#` required.